### ISSUE TITLE
Allow setting maxBodyLength

### ISSUE THREAD

**Issue by duyleicetea** [directus/sdk#56]
This is Axios Error but I think Directus should has solution for this.

![image](https://user-images.githubusercontent.com/106047176/189807589-f908f5b0-07d5-455a-904d-92ed1391ec14.png)

**Comment by azrikahar** [comment]
How are you making the request currently? Would you mind sharing a code snippet?

It should be possible to pass in the `maxBodyLength` to Axios via `requestOptions`, such as:

```js
await directus.files.createOne(formData, undefined, {
    requestOptions: {
        maxBodyLength: Infinity
    },
})
```

**Comment by duyleicetea** [comment]
```
  const form = new FormData()
    attachment.forEach((it) => form.append('file', it.buffer, `${game_id}-${faker.random.alphaNumeric(10)}`))

    const listFile = await this.directus.files.createOne(
      form,
      {},
      {
        requestOptions: {
          headers: form.getHeaders(),
          //@ts-ignore
          maxBodyLength: Infinity,
        },
      }
    )
    ```
maxBodyLength isn't in requestOptions type. I have to use `//@ts-ignore` but got the same error

**Comment by azrikahar** [comment]
Ahh... I was still thinking of a _very_ old sdk implementation which exposed the underlying axios instance, thus we were able to do something like:

```js
directus.transport.axios.post(
    `${baseUrl}/files`,
    formData,
    {
      headers: { ...headers, Authorization: `Bearer ${directus.auth.token}` },
      maxContentLength: Infinity,
      maxBodyLength: Infinity,
    }
  );
```

but it was removed in directus/directus#9080 in favor of recommending using axios directly. As of now, you will have to either use axios directly (technically it's already a dependency anyway) and do a similar approach as shown above to attach the auth token, or possibly look into a custom transport as described here: https://docs.directus.io/reference/sdk.html#transport

**Comment by azrikahar** [comment]
Resolved by #91.
